docs/developer-docs/latest/setup-deployment-guides/deployment/strapi-cloud.md

```diff
@@ -0,0 +1,119 @@
+    * **GitHub branch**: The default branch to use for this deployment. This uses the [default branch](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-branches-in-your-repository/changing-the-default-branch) of the repository but can be changed via the drop-down.
+    * Deploy on push: When enabled, this will automatically deploy the latest changes from the selected branch. When disabled, you will need to manually deploy the latest changes.
+
+![Project Setup](../assets/deployment/cloud/setup.png)
+
+4. (**Optional**) Select **Show Advanced Settings** to configure the following options:
+    * [**Environment variables**](../configurations/optional/environment.md): Environment variables are used to configure your Strapi app.
+    * **Base directory**: The directory where your Strapi app is located in the repository. This is useful if you have multiple Strapi apps in the same repository.
+
+![Advanced Setup](../assets/deployment/cloud/advanced.png)
+
+5. Click **Create** to finalize the project creation. An initial deployment is triggered and you are redirected to the **Projects** page.
+
+## Managing projects
+
+The **Projects** page displays a list of all your Strapi Cloud projects. From here you can manage your projects and access the corresponding applications.
+
+![Projects page - List](../assets/deployment/cloud/project_list.png)
+
+Each project card displays the following information:
+
+* Project name
+* Status: Displays a **Disconnected** warning if the project repository is not connected to Strapi Cloud.
+* Last deployment date: Timestamp of the last deployment.
+
+## Project details
+
+From the **Projects** page, click on any project card to access that project's details page.
+
+The project details page displays the following tabs: **Deploys** and **Settings**.
+
+### Deploys
+
+The **Deploys** tab displays a chronological list of cards with the details of all historical deployments for the project.
+
+![Project deploys](../assets/deployment/cloud/deploys.png)
+
+Each card displays the following information:
+
+* Commit SHA
+* Commit message
+* Deployment status: Whether the project is **Deploying**, the **Build failed**, or the deployment is **Done**.
+* Last deployment time: When the deployment was triggered and the duration.
+* Production branch
+* Options menu (`...`): The available options vary depending on the deployment status.
+    * For **Done** status: No further options.
+    * For **Deploying** status, you can:
+        * **Cancel deploy**
+    * For **Build failed** status, you can:
+        * **Download error logs**
+
+From this page you can also trigger a new deployment and access the application using the corresponding buttons.
+
+## Settings
+
+The **Settings** enables you to edit the following details for the project:
+
+* **Project name**: The name of your Strapi app, used to identify the project on the Cloud Dashboard, Strapi CLI, and deployment URLs.
```

Comment:
I think there's something with markdown here (bullet points are not displayed properly [on staging](https://documentation-46zoz9e3p-strapijs.vercel.app/developer-docs/latest/setup-deployment-guides/deployment/strapi-cloud.html)). Thanks!